web/app/package.json

```diff
@@ -45,6 +44,7 @@
     "eslint-plugin-promise": "^3.6.0",
     "eslint-plugin-react": "^7.5.1",
     "file-loader": "^1.1.5",
+    "karma": "^1.7.1",
```

Comment:
unintended change? I'd revert this file

Reply:
yes, totally accidental. thanks for catching it!